src/soundio/soundmanager.h

```diff
@@ -111,6 +113,15 @@ class SoundManager : public QObject {
         return m_pNetworkStream;
     }
 
+    void underflowHappened(int code) {
+        m_underflowHappened = 1;
+        if (CmdlineArgs::Instance().getDeveloper()) {
```

Comment:
Should we always write this to the log? It could help with user support if we could see that in the log.

Reply:
This would make the situation worse, because we are here in the engine code and a Logging is locking code. A small unhearable glitch can become hearable because of that. 
I will drop a comment.